agent/functional_tests/tests/functionaltests_unix_test.go

```diff
@@ -668,6 +668,54 @@ func TestAgentIntrospectionValidator(t *testing.T) {
 	assert.Equal(t, 42, exitCode, fmt.Sprintf("Expected exit code of 42; got %d", exitCode))
 }
 
+func TestRunAWSVPCTaskWithENITrunkingEndPointValidation(t *testing.T) {
+	RequireDockerVersion(t, ">=17.06.0-ce")
+
+	os.Setenv("ECS_FTEST_FORCE_NET_HOST", "true")
+	agent := RunAgent(t, &AgentOptions{
+		EnableTaskENI: true,
+		ExtraEnvironment: map[string]string{
+			"ECS_ENABLE_TASK_IAM_ROLE": "true",
+			"ECS_ENABLE_HIGH_DENSITY_ENI": "false",
```

Comment:
shouldn't this be true?

Reply:
This will be set to true in final testing after the backend changes are deployed. Added to a TODO note.